app/src/main/AndroidManifest.xml

```diff
@@ -99,6 +99,7 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.settings.profile.ProfileRenameActivity"
+      android:label="@string/profile_rename_title"
```

Comment:
Rename to `profile_rename_activity_title ` ... also update it wherever this string is getting used.